src/sentry/api/endpoints/project_commits.py

```diff
@@ -0,0 +1,42 @@
+from rest_framework.request import Request
+from rest_framework.response import Response
+
+from sentry.api.bases.project import ProjectEndpoint, ProjectReleasePermission
+from sentry.api.exceptions import ResourceDoesNotExist
+from sentry.api.serializers import serialize
+from sentry.models import ReleaseCommit
+from sentry.ratelimits.config import SENTRY_RATELIMITER_GROUP_DEFAULTS, RateLimitConfig
+
+
+class ProjectCommitsEndpoint(ProjectEndpoint):
+    permission_classes = (ProjectReleasePermission,)
+    rate_limits = RateLimitConfig(
+        group="CLI", limit_overrides={"GET": SENTRY_RATELIMITER_GROUP_DEFAULTS["default"]}
```

Comment:
This would mean that GET Project Commits has the global default 40 request per second rate limit instead of CLI default of 100 requests per second.

Reply:
I used the `ProjectReleasesEndpoint` as a model for the `ProjectCommitsEndpoint` since I'll be using the `ProjectCommitsEndpoint` to do something very similar on the frontend, so I applied the same rate limit for now.

Reply:
Ah yes, that was because we only wanted to put `ProjectReleasesEndpoint.post` into the CLI group, while leaving `ProjectReleasedEndpoint.get` with the default. Since there isn't another http method in this endpoint, you could just omit `rate_limits` entirely, and it'll fall back on the default. You could also just leave it as is, it isn't a blocker for this PR.

Either way, I realize the override is a bit confusing. I'll try to improve it.

Reply:
Thanks for clarifying!